portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,32 @@
 #content {
+  margin-top: 50px;
   margin-left: auto;
   margin-right: auto;
   width: 650px;
```

Comment:
I notice you have a specific width set on several content divs. My usual strategy, which you might find helpful, is to restrict the body element to this width, then let content divs be 100% (the default), which means they're 100% of their body parent.

Reply:
Could I make this change without it affecting the nav bar?